components/compliance-service/api/tests/Gemfile

```diff
@@ -6,4 +6,5 @@ gem 'rb-readline', '~> 0.5'
 gem 'minitest-hooks', '~> 1.5.0'
 gem 'grpc', '= 1.18.0'
 gem 'grpc-tools', '= 1.18.0'
-gem 'google-protobuf', '= 3.6.1'
+gem 'google-protobuf', '= 3.8.0'
```

Comment:
There are a lot of changes in the tests but no app updates, is updating this gem the reason we had to make so many changes? Is there any particular reason we need to update this gem?

Reply:
aren't most of the changes a result of updating these to be json?

Reply:
Latest ruby version from chefdk didn't work with google-protobuf 3.6.1.
Some of the tests updated are fixes to tests that were ignored due to a testing framework bug Rick is working on.

Reply:
the 'exit' bug was taken care of in this pr. 
7aa54818ac797c60855b913a6cf23859b39b7d8f